project_security_group/security/project_security_group_security.xml

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+    <!-- GROUPS -->
+
+    <!-- New group Project Responsible -->
+    <record id="group_project_responsible" model="res.groups">
+        <field name="name">Responsible</field>
+        <field name="category_id" ref="base.module_category_project_management"/>
+        <field name="implied_ids" eval="[(6, 0, [ref('project.group_project_user')])]"/>
+    </record>
+
+    <!-- New group End User -->
+    <record id="group_project_end_user" model="res.groups">
+        <field name="name">End User</field>
+        <field name="category_id" ref="base.module_category_project_management"/>
+    </record>
+
+    <!-- Rename group Project Manager to Project Administrator -->
+    <record id="project.group_project_manager" model="res.groups">
+        <field name="name">Administrator</field>
+        <field name="implied_ids" eval="[(6, 0, [ref('group_project_responsible')])]"/>
+    </record>
+
+    <!-- Modify implied_ids -->
+    <record id="project.group_project_user" model="res.groups">
+        <field name="implied_ids" eval="[(6, 0, [ref('group_project_end_user')])]"/>
+    </record>
+
+</data>
+
+<data noupdate="1">
+
+    <!-- PROJECT RULES -->
+
+    <!-- NEW Project Responsible rule-->
+    <record id="project_project_responsible_rule" model="ir.rule">
+        <field name="name">Project: project responsible: manage assigned projects</field>
+        <field name="model_id" ref="project.model_project_project"/>
+        <field name="domain_force">[
+            '|','|','|',
+                ('user_id', '=', user.id),
+                ('privacy_visibility', 'in', ['public', 'portal', 'employees']),
+                ('message_follower_ids', 'in', [user.partner_id.id]),
+                ('members', 'in', [user.id]),
+        ]</field>
```

Comment:
My idea is that I would only manage the projects I am responsible for.
If I'm in another Projet's team, I can't manage it.
If so, the domain should be just `('user_id', '=', user.id)`.